src/openzaak/components/catalogi/tests/test_informatieobjecttype.py

```diff
@@ -740,6 +741,28 @@ def test_filter_geldigheid(self):
         self.assertEqual(len(data), 1)
         self.assertEqual(data[0]["url"], f"http://testserver{reverse(iotype)}")
 
+    @override_settings(ALLOWED_HOSTS=["openzaak.nl"])
+    def test_filter_zaaktype(self):
```

Comment:
Could you add a test to check the behavior in case the zaaktype doesn't exist? E.g. the URL does not point to a zaaktype in the db

Reply:
The test is added